src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs

```diff
@@ -362,6 +361,8 @@ public override void Initialize([AllowNull] ViewWithData data)
             }
 
             IsLoading = IsBusy = false;
+            IsBusy = false;
+            usageTracker.IncrementPullRequestOpened().Forget();
```

Comment:
This method actually gets called when the pane is refreshed too, we probably don't want to increment the counter there. I think you can tell if it's a new pull request when `Model == null` on entry to the method. So I'd set a flag there and only increment if the flag is set.